SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -182,6 +182,10 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 	void TrackTombstone(const Worker_EntityId EntityId);
 #endif
 
+	// IsReady evaluates the GSM, Load Balancing system, and others to get a holistic
+	// view of whether the SpatialNetDriver is ready to assume normal operations.
```

Comment:
Can we make it explicit that this is for server workers only? (Or alternatively, add a similar setting to FindAndDispatchStartupOpsClient)

Reply:
Oh it does